src/main/java/org/broadinstitute/consent/http/enumeration/EmailType.java

```diff
@@ -16,7 +16,8 @@ public enum EmailType {
   NEW_RESEARCHER(12),
   RESEARCHER_APPROVED(13),
   NEW_DATASET(14),
-  NEW_DAA_REQUEST(15);
+  NEW_DAA_REQUEST(15),
+  NEW_DAA_UPLOAD(16);
```

Comment:
Consider splitting this into `NEW_DAA_UPLOAD_RESEARCHER` and `NEW_DAA_UPLOAD_SO` to be able to more easily differentiate it when auditing emails or logs.